src/CharterManager.sol

```diff
@@ -282,10 +289,12 @@ contract CharterManagerImp {
 
     function onVatFlux(address gemJoin, address from, address to, uint256 wad) external {}
 
-    function quit(bytes32 ilk, address u, address dst) external allowed(u) {
+    function quit(bytes32 ilk, address u, address dst) external allowed(u) allowed(dst) {
```

Comment:
Why is `allowed(dst)` added? It seems to me that you can safely send your urn to whoever you want.

Reply:
see above.